### PR TITLE
BusinessFacade implementation

### PR DESCRIPTION
**Pull request by AndreasBoegh**

### CODE REVIEW

File: HospitalGame_Business/src/business/BusinessFacade.java [hunk 1]
```diff
@@ -5,128 +5,206 @@
...
+        npcFacade = new NPCFacade();
+        map.injectItemFacade(itemFacade);
+        map.InjectNPCFacade(npcFacade);
+        
+    }
+
+    public final ArrayList<INPC> makeUpNpcs() {
+        ArrayList<INPC> npcs = new ArrayList<>();
+        Computer computer = new Computer("computer", false, null, NPCID.COMPUTER);
+        Porter porter = new Porter("porter", false, null, NPCID.PORTER);
```
Comment (larsbjn) [inline HospitalGame_Business/src/business/BusinessFacade.java]: Start names with big letter and porter isMoveable

File: HospitalGame_Business/src/business/BusinessFacade.java [hunk 2]
```diff
@@ -5,128 +5,206 @@
...
+import java.util.Arrays;
+import java.util.Random;
 
 /**
  *
  * @author andreasmolgaard-andersen
  */
 public class BusinessFacade implements IBusiness {
 
+    private ArrayList<IItem> items = new ArrayList<>();
```
Comment (larsbjn) [inline HospitalGame_Business/src/business/BusinessFacade.java]: What does this do?

**Review by larsbjn — CHANGES_REQUESTED** [review]

File: HospitalGame_Business/src/business/BusinessFacade.java [hunk 3]
```diff
@@ -5,128 +5,206 @@
...
+    public BusinessFacade() {
+        map = new Map();
+        itemFacade = new ItemFacade();
+        npcFacade = new NPCFacade();
+        map.injectItemFacade(itemFacade);
+        map.InjectNPCFacade(npcFacade);
+        
+    }
+
+    public final ArrayList<INPC> makeUpNpcs() {
```
Comment (frederikrosenberg) [inline HospitalGame_Business/src/business/BusinessFacade.java]: What is this used for?
And if it is used in the facade only it should be _private_

**Review by frederikrosenberg — CHANGES_REQUESTED** [review]

File: HospitalGame_Business/src/business/BusinessFacade.java [hunk 4]
```diff
@@ -24,109 +40,169 @@
...
+    public BusinessFacade() {
+        map = new Map();
+        itemFacade = new ItemFacade();
+        npcFacade = new NPCFacade();
+        map.injectItemFacade(itemFacade);
+        map.InjectNPCFacade(npcFacade);
+        
+    }
+
+    private final ArrayList<INPC> makeUpNpcs() {
```
Comment (larsbjn) [inline HospitalGame_Business/src/business/BusinessFacade.java]: delet dis

**Review by larsbjn — CHANGES_REQUESTED** [review]

**Review by larsbjn — APPROVED** [review]

**Review by frederikrosenberg — APPROVED** [review]

**Pull request merged**